docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -216,6 +235,51 @@ $ kubectl create namespace teleport
 $ kubectl --namespace teleport create -f aws-issuer.yaml
 ```
 
+## Step 4b. Use ACM to handle TLS
+
+To use ACM to handle TLS, add additional annotations to the chart specifying the ACM certificate ARN to use.
+
+We also use the `extraArgs` value to disable Teleport's TLS listener, making the backend connection use plain TCP from the load balancer
```

Comment:
```suggestion
Include the `extraArgs` value to disable Teleport's TLS listener, making the backend connection use plain TCP from the load balancer
```
To make this consistent with the surrounding text

Reply:
Also, I think we should expand on this paragraph. We haven't introduced "the load balancer" yet, so we should indicate here that the Teleport Auth/Proxy Services in the `teleport-cluster` chart use a load balancer service. 

I'm also not sure what "the backend connection" means.